R/human_infection.R

```diff
@@ -145,14 +181,16 @@ calculate_infections <- function(
 #' @param renderer model render object
 #' @param parameters model parameters
 #' @param prob vector of population probabilities of infection
+#' @param relative_rates relative rates of hypnozoite relapse relative to total infection rate
 #' @noRd
 infection_outcome_process <- function(
```

Comment:
Again, this function is quite long. It would be cleaner as `falciparum_infection_outcome` and `vivax_infection_outcome`.